test/Generators/Microsoft.Gen.Logging/TestClasses/MessageTestExtensions.cs

```diff
@@ -2,46 +2,38 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using Microsoft.Extensions.Logging;
-using Microsoft.Extensions.Telemetry.Logging;
 
 namespace TestClasses
 {
     internal static partial class MessageTestExtensions
     {
-        [LogMethod(0, LogLevel.Trace, null!)]
+        [LoggerMessage(0, LogLevel.Trace, null!)]
         public static partial void M0(ILogger logger);
 
-        [LogMethod(1, LogLevel.Debug, "")]
+        [LoggerMessage(1, LogLevel.Debug, "")]
         public static partial void M1(ILogger logger);
 
-        [LogMethod(2, LogLevel.Debug)]
+#pragma warning disable LOGGEN030
+        [LoggerMessage(LogLevel.Debug)]
         public static partial void M2(ILogger logger);
+#pragma warning restore LOGGEN030
 
-#if false
-
-        // These are disabled due to https://github.com/dotnet/roslyn/issues/52527
-        //
-        // These are handled fine by the logger generator and generate warnings as expected. Unfortunately, the above warning suppression is
-        // not being observed by the C# compiler at the moment, so having these here causes build warnings.
-
-        [LogMethod(2, LogLevel.Trace)]
+        [LoggerMessage(LogLevel.Trace)]
         public static partial void M2(ILogger logger, string p1, string p2);
 
-        [LogMethod(3, LogLevel.Debug, "")]
+        [LoggerMessage(LogLevel.Debug, "")]
         public static partial void M3(ILogger logger, string p1, int p2);
 
-        [LogMethod(4, LogLevel.Debug, "{p1}")]
+        [LoggerMessage(LogLevel.Debug, "{p1}")]
```

Comment:
I assume this is why you changed what I asked in my previous question where we are no longer setting an ID on the logger?

Reply:
Yep.